src/utils/update/managing.officer.mapper.ts

```diff
@@ -69,6 +71,13 @@ export const splitNames = (officerName: string): string[] => {
   }
 };
 
+export const splitNationalities = (officerNationalities: string | undefined): string[] => {
+  if (!officerNationalities) {
+    return [ "" ];
+  }
+  return officerNationalities.length > 1 ? officerNationalities.split(",") : [officerNationalities];
```

Comment:
Not sure why you would special case if the officersNationalities is longer than 1 character? Do you have a test for that not being the case? Or just drop that test?

Reply:
Do you need to trim values if it "country1, country2"?